README.md

```diff
@@ -37,6 +37,7 @@ Once you have promtail, Loki, and Grafana running, continue with [our usage docs
 - [API documentation](./docs/api.md) for alternative ways of getting logs into Loki.
 - [Operations](./docs/operations.md) for important aspects of running Loki.
 - [Promtail](./docs/promtail-setup.md) on how to configure the agent that tails your logs.
+- [Logcli](./docs/Logcli.md) on how to query your logs without Grafana.
```

Comment:
It looks like the path here should be `./docs/logcli.md` with a lower case L
Thanks!